docs/workflows.rst

```diff
@@ -25,7 +25,7 @@ is presented below:
                                 longitudinal=False,
                                 t2s_coreg=False,
                                 omp_nthreads=1,
-                                freesurfer=True,
+                                reconall=True,
```

Comment:
I would leave these be for now. (I wouldn't use `reconall` anyway, since turning off `freesurfer` also turns off bbregister and surface sampling). Once we decide the final form of this option will be time enough to change parameter names. (Users don't see this, anyway.)